src/XmlMiddleware.php

```diff
@@ -13,10 +13,10 @@ class XmlMiddleware
     const MIME_TYPE_XML = 'text/xml';
     const OPTION_XML = 'xml';
 
-    public static function xml()
+    public static function xml(string $xmlEncoding = 'utf-8')
```

Comment:
Replace `string $xmlEncoding = 'utf-8'` with `array $context = ['xml_encoding' => 'utf-8']`, but also I think `xml_encoding` is not necessary, because `xml_encoding` has `utf-8` as default.
![image](https://user-images.githubusercontent.com/2156468/125866274-7a89ec1b-8b3e-4f7d-baee-b6159c1977c2.png)

Reply:
Thank you your feed back. I has read this document, but I always send non utf8 request out.
When I add xml_encoding, it will send utf8 xml request.